client/blocks/installments-plan-picker/index.jsx

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+export class InstallmentsPlanPicker extends React.Component {
+	static propTypes = {
+		initialValue: PropTypes.number,
+		plans: PropTypes.array.isRequired,
+		onChange: PropTypes.func,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		onChange: () => null,
+	};
+
+	state = {
+		installmentsInputValue: this.props.initialValue,
+	};
+
+	formatPlans() {
+		return this.props.plans.map( plan => {
+			plan.label = plan.value + ' x ' + plan.payment;
+			return plan;
+		} );
+	}
+
+	handleInstallmentsInput = event => {
+		const installments = parseInt( event.target.value );
+
+		this.setState( {
+			installmentsInputValue: installments,
+		} );
+
+		this.props.onChange( installments );
+	};
+
+	render() {
+		const installmentPlans = this.formatPlans();
+		return (
+			<div className="installments-plan-picker">
+				<div className="installments-plan-picker__header">
+					{ this.props.translate( 'Choose an installment plan', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Choose an installments plan' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).